tools/clang/test/HLSLFileCheck/pix/DynamicSamplerOutOfBounds.hlsl

```diff
@@ -21,4 +21,4 @@ float4 Main() : SV_Target
 
 // Out of bounds sampler access should be at offset 512
 // CHECK: call void @dx.op.bufferStore.i32(
-// CHECK:i32 512, i32 undef, i32 16777216
+// CHECK:i32 512, i32 undef, i32 134217728
```

Comment:
Note: you can use `CHECK-SAME:` to check something on the same line that the prior `CHECK:` matched.  This can help eliminate both accidental matches, and make it clear where the snippet is supposed to be to future editors of the test.

Reply:
Thanks for the tip!